other/upgrade.php

```diff
@@ -5098,7 +5098,7 @@ function MySQLConvertOldIp($targetTable, $oldCol, $newCol, $limit = 50000, $setS
 		{
 			$arIp[$i] = trim($arIp[$i]);
 
-			if (empty($arIp[$i]))
+			if (!filter_var($arIp[$i], FILTER_VALIDATE_IP, FILTER_FLAG_IPV4))
```

Comment:
I think we should allow for IPV4 & IPV6 here

Reply:
```suggestion
			if (!filter_var($arIp[$i], FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 | FILTER_FLAG_IPV6))
```

Reply:
why the newline giithub why u ignore our conffig